### PR TITLE
docs(pages): update Japanese translation of env-variables.mdx

### PR DESCRIPTION
**Pull request by uehaj**
update Japanese translation of env-variables.mdx for commit 5cef63a7ae0c86cc1a8787208067d7064cf2d745

Closes #<issue>

## ✅ Checklist

- [v] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [v] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

update Japanese translation of env-variables.mdx

### CODE REVIEW

**Comment by c-ehrlich** [comment]
cc @t6adev

**Comment by uehaj** [comment]
Hi, is there some problems for this PR?

**Comment by c-ehrlich** [comment]
> Hi, is there some problems for this PR?

- it needs review from @t6adev or another japanese speaker
- please remove the changeset - i know CI told you to add it, this is a mistake on our side. we'll get the github action fixed soon.

**Review by c-ehrlich — APPROVED** [review]
It hasn't been reviewed by another native speaker, but it's definitely better than before so let's get this merged :)

File: .changeset/config.json
Comment (juliusmarminge) [inline .changeset/config.json:8]: Woooah whats happening here

**Comment by c-ehrlich** [comment]
sorry, we had to revert this because it changed `.changeset/config.json` and some other stuff, i somehow missed this in my final review. 

please feel free to open a new pr :)

**Pull request merged**
